tests/test_ndb_json.py

```diff
@@ -101,6 +100,8 @@ def test_loads_with_naive_values(self):
 
     def test_loads_with_date_and_time_values(self):
         """Assert that date/time-like strings are parsed properly."""
+        assert datetime.datetime(2015, 1, 1) == ndb_json.loads(datetime.date(2015, 1, 1).isoformat())
+
         payload_str = json.dumps({
             'date': datetime.date(2015, 1, 1).isoformat(),
             'datetime': datetime.datetime(2015, 1, 1, 12, 0, 0, 0).isoformat(),
```

Comment:
I think it'd be good to nest a datetime another level to test that functionality, e.g. add a key like:
`'nested_datetime': {'datetime': datetime.datetime(2015, 1, 3)}`